avm/res/compute/virtual-machine/tests/e2e/windows.nvidia/main.test.bicep

```diff
@@ -11,8 +11,8 @@ metadata description = 'This instance deploys the module for a VM with dedicated
 @maxLength(90)
 param resourceGroupName string = 'dep-${namePrefix}-compute.virtualMachines-${serviceShort}-rg'
 
-@description('Optional. The location to deploy resources to.')
-param resourceLocation string = deployment().location
+@description('Optional. The location to deploy resources to.') // required because Test-TemplateDeployment.ps1 requires the resourceLocation parameter, value must equal to tempLocation
```

Comment:
Similar suggestion as per the windows hostpool test

Reply:
Done!